spec/requests/resource_operations_spec.rb

```diff
@@ -70,7 +70,7 @@
   end
 
   describe 'POST /articles' do
-    subject(:last_response) { post("/articles", '{ "data": { "type": "articles" } }') }
+    subject(:last_response) { post("/articles", '{ "data": { "id": "1", "type": "articles" } }') }
```

Comment:
Yeah I'm fine with this :+1: It didn't come to my mind that it could be set on the database side, too. I agree it's not necessary to complicate the test database setup with ID generation, so it's ok that the ID is specified here.

Should you, though, use another id as `1` here? Doesn't the fixtures already specify an article with the similar ID? I wonder why this request didn't fail if there already was an article in the DB :confused: